### PR TITLE
GH-2009 Update to java 13 for Github Actions

### PR DESCRIPTION
**Pull request by hmottestad**
GitHub issue resolved: #2009  

Briefly describe the changes proposed in this PR:

* bumps build on Java 11 to Java 13
* bumps version of elasticsearch (see also #2016 )
* fixes issue in SPARQLStoreConnectionTest with use of reflection

### CODE REVIEW

**Comment by hmottestad** [comment]
Not sure why we now have test failures:

```
2020-03-18T10:25:11.8766472Z Tests in error: 
2020-03-18T10:25:11.8767756Z   SPARQLStoreConnectionTest.testGetStatementsContextHandling:178->enableQuadModeOnConnection:217 Â» NoSuchField
2020-03-18T10:25:11.8770797Z   SPARQLStoreConnectionTest.testGetStatementsContextHandling:178->enableQuadModeOnConnection:217 Â» NoSuchField
2020-03-18T10:25:11.8773956Z   SPARQLStoreConnectionTest.testGetStatementsContextHandling:178->enableQuadModeOnConnection:217 Â» NoSuchField
2020-03-18T10:25:11.8776936Z   SPARQLStoreConnectionTest.testGetStatementsContextHandling:178->enableQuadModeOnConnection:217 Â» NoSuchField
2020-03-18T10:25:11.8780727Z   SPARQLStoreConnectionTest.testGetStatementsContextHandling:178->enableQuadModeOnConnection:217 Â» NoSuchField
2020-03-18T10:25:11.8784118Z   SPARQLStoreConnectionTest.testGetStatementsContextHandling:178->enableQuadModeOnConnection:217 Â» NoSuchField
2020-03-18T10:25:11.8787035Z 
2020-03-18T10:25:11.8787192Z Tests run: 1439, Failures: 0, Errors: 6, Skipped: 99
```

**Comment by jeenbroekstra** [comment]
Not entirely sure what is going on, but that test failure is in a place in the code where there is some pretty dirty reflection used to tweak the behavior of the SPARQLConnection.

**Comment by jeenbroekstra** [comment]
I've got a fix for this, I'll push it to this branch.

**Comment by jeenbroekstra** [comment]
@hmottestad assuming this fixes the test failure, could you clean up the commits a bit on this branch before merging?

File: core/sail/elasticsearch-store/src/test/java/org/eclipse/rdf4j/sail/elasticsearchstore/TestHelpers.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
 public class TestHelpers {
 
 	private final static Random random = new Random();
-	public static final String VERSION = "6.5.4";
+	public static final String VERSION = "6.8.7";
```
Comment (jeenbroekstra) [inline core/sail/elasticsearch-store/src/test/java/org/eclipse/rdf4j/sail/elasticsearchstore/TestHelpers.java]: Is this a change we need? From your comments I got the impression you merely tried to use this to fix some test failures - or were those _different_ test failures? 

If we don't need this for Java 13, let's roll it back so we can merge this independently of the Elasticsearch version bump.

**Comment by jeenbroekstra** [comment]
@hmottestad the version bumps of Lucene etc have been merged. To be safe, could you rebase this and see if all builds and tests as expected?

**Pull request merged**